ouroboros-consensus/src/Ouroboros/Storage/ChainDB/Impl/Reopen.hs

```diff
@@ -144,7 +144,7 @@ reopen (CDBHandle varState) launchBgTasks = do
           cdbEpochInfo
           immDbTipPoint
           (contramap TraceLedgerReplayEvent cdbTracer)
-        LgrDB.reopen cdbLgrDB cdbImmDB lgrReplayTracer
+        replayed <- LgrDB.reopen cdbLgrDB cdbImmDB lgrReplayTracer
```

Comment:
Line 80 of this file needs to be removed: it takes a snapshot on shut down, which is exactly a bug this PR is solving.